### PR TITLE
Maintenance: Changes for Zammad 5.2.

### PR DESCRIPTION
**Pull request by mgruner**
Update to Ruby 3.0.3.

### CODE REVIEW

**Comment by mgruner** [comment]
This should be merged after Zammad 5.2 is released.

**Review by MrGeneration — DISMISSED** [review]

**Review by monotek — APPROVED** [review]
Would be nice the ci stuff (even unrelated) could be fixed too.

**Comment by MrGeneration** [comment]
@monotek I'll be preparing a PR outside of this PR to allow MG rebasing so that we don't have to do this multiple times.

**Comment by mgruner** [comment]
@monotek @MrGeneration thanks for the linter fixes. This PR is now rebased.

File: containers/zammad/Dockerfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-      org.label-schema.schema-version="5.1.0" \
+      org.label-schema.schema-version="5.2.0" \
```
Comment (mgruner) [inline containers/zammad/Dockerfile:40]: @monotek the change for the Zammad version label to 5.2 is also added now.

**Review by monotek — APPROVED** [review]

**Pull request merged**